Debugging/NodeJs.md

```diff
@@ -0,0 +1,29 @@
+
+# NodeJs
+
+Installation instructions of the Tessel CLI can be found here: http://tessel.github.io/t2-start/
+
+This document will address common problems related to NodeJs usage.
+
+## NVM
+
+The official installation instructions assume Node is installed globally. For those of you using NVM this will not be the case. As indicated [here](https://github.com/creationix/nvm/issues/786) NVM does not install globally by design.
+
+### Linux
+
+Install `t2-cli` module as normal. To install the drivers, perform the following:
```

Comment:
Can we add a bit more information here about why the drivers need to be installed separately? Maybe @johnnyman727 can provide some context around that.